packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -675,6 +748,29 @@ class Scaffold extends StatefulWidget {
     return scaffoldScope.geometryNotifier;
```

Comment:
Add BottomAppBar to the list of "See also"s for Scaffold around line 500.

Reply:
done